### PR TITLE
[BACKLOG-36954] - Changed dialogs to responsive and removed fixed lengths

### PR DESCRIPTION
**Pull request by jsaraf**
Please review @pentaho/wcag

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 7m 9s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **398**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-875/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by dcleao — CHANGES_REQUESTED** [review]

File: widgets/src/main/java/org/pentaho/mantle/client/dialogs/scheduling/ScheduleParamsDialog.java [hunk 1]
```diff
@@ -130,7 +133,7 @@ public void initDialog() {
     wizardDeckPanel.getElement().getParentElement().addClassName( "schedule-dialog-content" );
     wizardDeckPanel.getElement().getParentElement().removeClassName( "dialog-content" );
 
-    setSize( "650px", "450px" );
+//    setSize( "650px", "450px" );
```
[the remark this answers is not in the document]
Comment (jsaraf) [reply widgets/src/main/java/org/pentaho/mantle/client/dialogs/scheduling/ScheduleParamsDialog.java:136]: Are you referring to `wizardDeckPanel.getElement().getParentElement().removeClassName( "dialog-content" );` or `setSize( "650px", "450px" );`

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 7m 3s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **398**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-875/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: widgets/src/main/java/org/pentaho/mantle/client/dialogs/scheduling/ScheduleParamsDialog.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (dcleao) [reply widgets/src/main/java/org/pentaho/mantle/client/dialogs/scheduling/ScheduleParamsDialog.java:136]: But the one you pointed out, yes — the class name should not be removed as well.
But I now realize an important issue — that not removing the class name may break other themes...
But I see not way but to try to fix those other themes and make them work despite the `dialog-content` class being there.

File: widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css [hunk 2]
```diff
@@ -597,15 +597,21 @@ div#promptPanel div.prompt-panel {
...
 div.parameter div.parameter-label {
   margin-bottom: 5px;
 }
 
 div#promptPanel div.parameter {
-  margin-bottom: 20px;
+  margin-bottom: unset;
+}
+
+.parameter-container > tbody > tr > td > div > table > tbody {
```
Comment (NJtwentyone) [inline widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css:611]: If not possible in our limited time, you don't have to address my comments.

But the css selector` tbody > tr > td > div > table > tbody` seems really brittle. Could you use another css selector to target the html content?

I see other examples in the file such as`div#promptPanel div.parameter` and `div#promptPanel div.parameter`

**Pull request closed without merging**